database/mssql/scripts/versions/revert/v_6_ddl_revert.sql

```diff
@@ -0,0 +1,16 @@
+SET ANSI_NULLS ON
+GO
+SET QUOTED_IDENTIFIER ON
+GO
+SET NOCOUNT ON
+GO
+BEGIN TRANSACTION
+
+DROP TABLE [dms].[ORBC_DOCUMENT]
+
```

Comment:
Need to drop [dms] schema after dropping the document table as well

Reply:
Resolved